app/views/gardens/_garden.html.erb

```diff
@@ -1,15 +1,39 @@
 <div class="card my-3">
-  <div class="card-header">
-    <h2><%= link_to garden.name, garden_path(garden) %></h2>
-  </div>
-  <div class="card-body">
-    <h5 class="card-title">beds</h5>
-    <p class="card-text">TO DO: iterate over beds and display bed info</p>
-    <%= render "partials/modal_edit_garden", garden: garden %>
-    <%= link_to "delete garden",
+  <div class="card-header garden-header">
+    <div class="d-flex justify-content-between">
+      <div>
+        <h2><%= link_to garden.name, garden_path(garden) %></h2>
+      </div>
+      <div>
+        <%= render "partials/modal_edit_garden", garden: garden %>
+        <%= link_to ('<i class="fa-solid fa-trash-can"></i>').html_safe,
                 garden,
-                data: {turbo_method: :delete, turbo_confirm: "Are you sure?" },
-                class: "btn btn-danger"
-              %>
+                data: {turbo_method: :delete, turbo_confirm: "Are you sure you want to delete this garden?" },
+                class: "btn btn-light"
+                %>
+      </div>
+    </div>
+  </div>
+  <div class="card-body garden">
+    <div class="d-flex flex-wrap justify-content-around">
+      <% garden.beds.each do |bed|%>
+        <div class="m-2">
+          <span class="badge badge-primary mb-1"><%= bed.description %></span>
+          <div class="card bed">
+            <div class="card-body">
+              <div class="d-flex flex-wrap justify-content-around">
+                <% bed.crops.each do |crop| %>
+                  <% 3.times do %>
+                    <h2 class="m-2" data-toggle="tooltip" data-placement="top" title="<%= crop.veggie.name %>">
+                      <%= crop.veggie.emoji %>
```

Comment:
nice! in the future we will be able to reference the crop emoji! all coming together 😎